Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -51,6 +51,24 @@ public override SizeRequest GetDesiredSize(int widthConstraint, int heightConstr
 		{
 			return new SizeRequest(new Size(Context.ToPixels(40), Context.ToPixels(40)));
 		}
+		
+		public virtual void OnMapReady(GoogleMap map)
```

Comment:
Would you prefer to have the protected virtual method take the GoogleMap parameter, or expect overrides to use the NativeMap property?